File: 백준/Bronze/32369. 양파 실험/양파 실험.c
#include <stdio.h>

void swap(int *a, int *b){
   int temp = *a;
   *a = *b;
   *b = temp;
}

int main(void) { 
   int a, b, c, good = 1, bad = 1;
   scanf("%d %d %d", &a, &b, &c);

   for(int i = 0; i < a; i++){
      good += b;
      bad += c;
      if(good < bad)   swap(&good, &bad);
      if(good == bad)   bad -= 1;
   }

   printf("%d %d", good, bad);

   return 0;
}

File: 백준/Bronze/32369. 양파 실험/양파 실험.py
a, b, c = map(int, input().split())

good, bad = 1, 1

for _ in range(a):
    good += b
    bad += c
    
    if good < bad:
        good, bad = bad, good
    elif good == bad:
        bad -= 1

print(good, bad)